src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -10,44 +10,48 @@ class SendEventEmailsJob < ApplicationJob
 
   def perform
     Event::Base.where(mails_sent: false).order(created_at: :asc).limit(1000).each do |event|
+      event.update_attributes(mails_sent: true)
```

Comment:
If something goes wrong below, you can never get back the event. Please do this as last action.

Reply:
Changed back to the `ensure` inside the `send_email` method.